src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -63,7 +64,13 @@ interface ParsedRow {
 // Helper -- Takes column header from file, converts to internal metadata key
 // If header is unrecognized, leaves it alone (useful for warnings, etc).
```

Comment:
I think it would be good to update these docs/comments to explain that users can provide headers as we give them in template from CZ GE, or they can also use the ones from Nextstrain as an alias.

Reply:
ok i will update the comment and am also updating the helper instructions text